style.css

```diff
@@ -0,0 +1,96 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  min-height: 100vh;
+  background-color: #d3d3d3;
+  font-family: cursive;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.cover {
+  margin: 0 auto;
+  width: 200px;
+  max-width: 800px;
+  min-width: 400px;
+  height: 480px;
+  background-image:;
+  background-color: #78aaeb;
+  border-radius: 10px;
+  box-shadow: 5px 10px 20px;
+  transform-origin: left;
+  animation: slide 5s ease-in-out normal forwards;
+  position: absolute;
+  z-index: 2;
+}
+::-webkit-scrollbar {
+  display: none;
+}
+.paper {
+  position: relative;
+  width: 50%;
+  max-width: 200px;
+  min-width: 400px;
```

Comment:
could you explain this, max-width: 200px  and min-width: 400px ? @nehalskdev

Reply:
Yes as I have given width of the div 50% a relative value, so it will occupy space in relative to its parent element. Max-width and min-width will maintain the size of div with respect to parent element. 
If you have better way to make me understand this I'm open for learning. 😁